qa/L0_onnxtrt_optimization/test.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
+# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Since it is new file, I think the copy right should be `# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.`

Reply:
Right. @askhade we have a new copyright format as above. Also new files should only have current year in copyright.

Reply:
Other, existing files that you modified, make sure 2021 is in copyright years and add "& AFFILIATES" to the line.